iceprod/server/data/www/rest.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Helper functions that use REST API
+ */
+
+async function fetch_json(url, passkey, json=null, post=false) {
```

Comment:
The `post` argument isn't very clear.  The first thing I thought was to enable POST vs regular GET, but that isn't what it does.

It might be a better idea to just ask for the method, and default to GET.  Then throw an error if we have json != null and method = GET.

Reply:
11b382e

changed fetch_jason()'s parameters to `fetch_json(method, url, json, passkey)`. Calls are now more verbose but hopefully more readable.